README.md

```diff
@@ -9,3 +9,53 @@ This will get the terraform mixin and install it from source.
 1. `go get -u get.porter.sh/mixin/terraform`
 1. `cd $(go env GOPATH)/src/get.porter.sh/mixin/terraform`
 1. `make build install`
+
+
+## Examples
```

Comment:
Can you add a section for outputs too that just mentions how it uses/connects to the outputs from terraform?